RealmSwift/Aliases.swift

```diff
@@ -93,16 +125,19 @@ extension ObjectBase {
 
      - warning: This method cannot be called during a write transaction, or when
                 the containing Realm is read-only.
-
+     - - parameter keyPaths: The object properties which trigger the block to
```

Comment:
Multiple dashes.

Reply:
Resolved in [b8e9ee0](https://github.com/realm/realm-cocoa/pull/7249/commits/b8e9ee02afed35f06f77bd688886ad6b7aea627d)